test_scripts/ATF_ShutDown_IGNITION_OFF.lua

```diff
@@ -0,0 +1,70 @@
+--  Requirement summary:
+--  Check that SDL finish its work properly by ignition off with one registered Application
+--
+--  Description:
+--  1. HMI is running
```

Comment:
@LitvinenkoIra, points:
1. HMI is running
2. One App is registered and activated on HMI
should be in section: "-- 1. Used preconditions(if applicable)"
For section: 
"-- 2. Performed steps" you should have actions:
Perform ignition Off
=> HMI sends OnExitAllApplications(IGNITION_OFF)

section: "-- Expected result:" should have:
1. SDL sends to App OnAppInterfaceUnregistered
2. SDL sends to HMI OnSDLClose and stops working